backend/service.edt/Kafka/ConsumerSetup.cs

```diff
@@ -19,34 +19,36 @@ public static IServiceCollection AddKafkaConsumer(this IServiceCollection servic
         var clientConfig = new ClientConfig()
         {
             BootstrapServers = config.KafkaCluster.BoostrapServers,
-            SaslMechanism = SaslMechanism.Plain,
+            SaslMechanism = SaslMechanism.OAuthBearer,
```

Comment:
We might want to make this configurable so for local setup/testing we can change the setup